File: Final Project/CommonLayer/CommonSite.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CommonLayer
{
     public class CommonSite
    {
        public int SiteId { get; set; }
        public string Name { get; set; }
        public string Area { get; set; }
        public string AddressGMaps { get; set; }
        public decimal Price { get; set; }
        public string Description { get; set; }
        public System.TimeSpan OpeningHour { get; set; }
        public System.TimeSpan ClosingHour { get; set; }
        public string Category { get; set; }
        public Nullable<System.TimeSpan> EstimatedStay { get; set; }

        public List<string> Equipment { get; set; }

        //public ICollection<CommonEquipmentSite> EquipmentSites { get; set; }
        //public ICollection<CommonTripSite> TripSite { get; set; }


    }
}

File: AngularProject/NofToursClient/src/app/models/site/commonSite.ts
export interface CommonSite
{
    Name: string;
    Area: string;
    AddressGMaps: string;
    Price: number;
    Description: string;
    OpeningHour:string;
    ClosingHour:string;
    Category:string;
    EstimatedStay:string;// in c# nullable timespan
    Equipment:string[];
    // equipment?
    // tripCommonSite?
}

//     constructor(
//        Name: string,
//         area: string,
//         addressGMaps:string,
//         description:string,
//         price:number,
//         openingHour:string,
//         closingHour:string,
//         category:string,
//         estimatedStay:string
//     )
//     {
//         this.name=name;
//         this.addressGMaps=addressGMaps;
//         this.category=category;
//         this.description=description;
//         this.estimatedStay=estimatedStay;
//         this.openingHour=openingHour;
//         this.closingHour=closingHour;
//         this.area=area;
//         this.price=price;
//     }

// }